### PR TITLE
Tools: Holybro S500 param file

### PR DESCRIPTION
**Pull request by rmackay9**
This is a parameter file for the [Holybro S500](http://www.holybro.com/product/s500-arf-v2-frame-kit/) to make setup easier.  I've tested this on a real vehicle with a [pix32v5 autopilot](https://ardupilot.org/copter/docs/common-holybro-pix32v5.html) and the setup was very easy and it flies nicely.

### CODE REVIEW

File: Tools/Frame_params/Holybro-S500.param [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+BATT_AMP_PERVLT,39.877
+BATT_CURR_PIN,1
+BATT_MONITOR,4
+BATT_VOLT_MULT,18.182
+BATT_VOLT_PIN,0
+BRD_SAFETYENABLE,0
+FRAME_CLASS,1
+FRAME_TYPE,1
+INS_GYRO_FILTER,40
+MOT_BAT_VOLT_MAX,12.6
```
Comment (tridge) [inline Tools/Frame_params/Holybro-S500.param:28]: the S500 web page seems to suggest 4S, but this is for 3S. Is that right?

Comment (rmackay9) [reply Tools/Frame_params/Holybro-S500.param:28]: yes, it is.  shoot..  I think I've got a 4S battery so I'll test with that.

Comment (tridge) [reply Tools/Frame_params/Holybro-S500.param:28]: might change tuning a bit too

Comment (rmackay9) [reply Tools/Frame_params/Holybro-S500.param]: updated using a 4S battery.

**Comment by rmackay9** [comment]
Re-tested and re-tuned using a 4S battery.  I'll merge after it passes CI.

**Pull request merged**